class AlreadyExistsError(Exception):
    pass

class GroupNotFullError(Exception):
    pass
